frb_rust/src/ffi/web.rs

```diff
@@ -54,6 +54,28 @@ impl IntoDart for chrono::Duration {
         self.num_milliseconds().into_dart()
     }
 }
+
+#[cfg(feature = "uuid")]
+impl IntoDart for uuid::Uuid {
+    #[inline]
+    fn into_dart(self) -> DartAbi {
+        self.as_bytes().to_vec().into_dart()
+    }
+}
+
+#[cfg(feature = "uuid")]
+impl IntoDart for Vec<uuid::Uuid> {
+    #[inline]
+    fn into_dart(self) -> DartAbi {
+        use std::io::Write;
+        let mut buffer = Vec::<u8>::with_capacity(self.len() * 16);
```

Comment:
nit: Again the "16" magic number

Reply:
resolved in 86edba18948fa2e64a68c0cad89bd96ad168f558